<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteDev::Internal::ProjectSettingsWidget</class>
 <widget class="QWidget" name="RemoteDev::Internal::ProjectSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="gbxMappings">
     <property name="title">
      <string>Mappings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tblMappings">
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbxSettings">
     <property name="title">
      <string>Settings for Mapping</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblDevice">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbxDevice"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPath">
        <property name="text">
         <string>Target path</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtPath"/>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkEnbled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>RemoteDev::Internal::ProjectSettingsWidget</receiver>
   <slot>newMapping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>RemoteDev::Internal::ProjectSettingsWidget</receiver>
   <slot>removeMapping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newMapping()</slot>
  <slot>removeMapping()</slot>
 </slots>
</ui>
